Display/Documentation/HowToTurnOn.md

```diff
@@ -0,0 +1,28 @@
+How to turn on the Tetra Ventillator Splitter
+=============================================
+
+1. Ensure that the splitter is connected to a power supply
+
+2. Ensure that the airlines connecting the splitter and ventillator are attached (detailed in a serperate document)
+
+3. Flip the on-off switch to on
+
+4. After a short boot-up sequence, a screen showing data for each of the patient airlines should appear, which looks like this
+
+   ![Airline data information screen](https://cdn.discordapp.com/attachments/718703783730741290/720436682851352697/VentilatorDisplay_1.jpeg)
+
+   - If instead a computer desktop appears, connect a mouse to the USB port on the splitter and double-click on the "Tetra Splitter Data Readout" icon
```

Comment:
We should probably make sure that this particular mode never ever happens

Reply:
Agree